plugin/src/main/resources/io/jenkins/plugins/casc/ConfigurationAsCode/index.jelly

```diff
@@ -54,7 +54,7 @@
     <h2>${%Reference}</h2>
     <dt>
       <dl><a href="reference">${%Documentation}</a></dl>
-      <dl><a href="schema">${%JSON schema}</a></dl>
+      <dl><a href="v2Schema">${%JSON schema}</a></dl>
```

Comment:
won't that send the user to a page which says they need to click a button to send it as a post request?

Reply:
Yeah it would because the method says `REQUIRE POST`

Reply:
is that required? cc @casz and @oleg-nenashev

Reply:
GET is fine here. As long as the call dies not modify the instance state, get requests are fine